go.mod

```diff
@@ -94,6 +94,7 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bombsimon/logrusr/v2 v2.0.1 // indirect
 	github.com/bradleyfalzon/ghinstallation/v2 v2.1.0 // indirect
+	github.com/caarlos0/env/v10 v10.0.0 // indirect
```

Comment:
please run `go mod tidy`, the linter show this dependence should be direct, with go mod tidy you can fix this 👌🏼